elevenclock/lang/lang_de.py

```diff
@@ -7,13 +7,13 @@
 # Meke special attention to elements like ":", etc.
 
 lang_2_8 = {
-    "Force the clock to be at the top of the screen": "",
-    "Show the clock on the primary screen": "",
-    "Use a custom font color": "",
-    "Use a custom background color": "",
-    "Align the clock text to the center": "",
-    "Select custom color": "",
-    "Hide the clock when a program occupies all screens": "",
+    "Force the clock to be at the top of the screen"            : "Uhr immer am oberen Rand des Bildschirms angezeigen",
+    "Show the clock on the primary screen"                      : "Uhr auf dem Hauptbildschirm angezeigen",
+    "Use a custom font color"                                   : "Eine benutzerdefinierte Schriftfarbe verwenden",
+    "Use a custom background color"                             : "Eine benutzerdefinierte Hintergrundfarbe verwenden",
+    "Align the clock text to the center"                        : "Ausrichten des Uhrentextes in der Mitte",
```

Comment:
Probably "Uhrzeit mittig ausrichten" would be a nice translation?

Reply:
I like it! Its changed